src/cpp/session/modules/rmarkdown/NotebookAlternateEngines.cpp

```diff
@@ -227,7 +227,7 @@ Error executeStanEngineChunk(const std::string& docId,
    
    // write code to file
    FilePath tempFile = module_context::tempFile("stan-", ".stan");
-   error = writeStringToFile(tempFile, code);
+   error = writeStringToFile(tempFile, code + "\n");
```

Comment:
The extra newline here is added to avoid errors that can occur during a call to `readLines(..., warn = TRUE)` -- I didn't hunt down where exactly that call was coming from however.